Mars Onboarding/StepDefinitions/LoginFeatureStepDefinitions.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using Mars_Onboarding.Pages;
+using OpenQA.Selenium;
+using Reqnroll;
+
+namespace Mars_Onboarding.StepDefinitions
+{
+    [Binding]
+    public class LoginFeatureStepDefinitions
+    {
+            private IWebDriver driver;
+
+            public LoginFeatureStepDefinitions(IWebDriver driver)
+            {
+                this.driver = driver;
+            }
+            LoginPage loginPage = new LoginPage();
```

Comment:
Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors